css_tasks/task11/js/popup.js

```diff
@@ -0,0 +1,52 @@
+import {elements} from './elements.js';
+
+export function popup() {
+  elements.then((data) => {
+    const {gallery, popup, body, overlay} = data;
```

Comment:
you can use destructuring in function variable ie ({gallery, popup, body, overlay}) => {}